tests/test_ruleset.py

```diff
@@ -94,6 +94,55 @@ def test_assert_multiple_facts():
     my_callback2.assert_called_with(result2)
 
 
+def test_squared_accessors():
```

Comment:
@tarilabs if you use the tools/convert_to_ast.py from the ansible-rulebook repo and convert the rulebook to ast format you will get the sources data included and then if we add the ast yml file to this list https://github.com/ansible/drools_jpy/blob/f1fade2146ca1fc3508a891ad69893016437acea/tests/test_ruleset.py#L935 it will validate all the rules. The integrated test knows how to insert the payload data from sources.

Reply:
Done with 324f2de